src/Microsoft.SqlTools.ServiceLayer/ShowPlan/ShowPlanGraph/Node.cs

```diff
@@ -55,6 +57,159 @@ public int GroupIndex
             get; set;
         }
 
+        /// <summary>
+        /// Gets Node display name
+        /// </summary>
+        public virtual string DisplayName
+        {
+            get
+            {
+                if (this.Operation == Operation.Unknown)
+                {
+                    return String.Empty;
+                }
+
+                // The display name can consist of two lines
+                // The first line is the Physical name and the physical kind in parenthesis
+                // The second line should contains either Object value or LogicalOp name.
+                // The second line should not show the same content as the first line.
+
+                string firstLine = this["PhysicalOp"] as string;
+                if (firstLine == null)
+                {
+                    if (this.Operation == null)
+                    {
+                        return String.Empty;
```

Comment:
nit: `string.Empty` is preferred to `String.Empty` per .Net Framework Coding Guidelines.  Basically primitive types should be preferred to BCL classes when both are available.  Probably not worth rewriting the code, more FYI for next time.